customer_outstanding_statement/__manifest__.py

```diff
@@ -12,7 +12,8 @@
     'website': 'https://github.com/OCA/account-financial-reporting',
     'license': 'AGPL-3',
     'depends': [
-        'account',
+        'account_invoicing',
```

Comment:
Also don't change account module to account_invoicing if strictly necessary.

This module is modifying things from account, not from account_invoicing.